semcore/table/src/style/table.shadow.css

```diff
@@ -114,11 +114,19 @@ SRow {
 SRow[theme='default'] {
   background-color: #fff;
 
-  &:hover SCellRow {
+  &:hover > SCellRow {
     background-color: color-mod(var(--stone) blend(#fff 90%));
   }
 }
 
+SRow[theme='default']:hover,
+SRow[theme]:hover SCellRow[theme='default'],
```

Comment:
maybe `SRow[theme]:hover SCellRow[theme='default']`

Reply:
![image](https://user-images.githubusercontent.com/81574652/112860424-79675a80-90bc-11eb-88e8-0e1f2eed5f1b.png)
I'll fix it